weed/filer/mongodb/mongodb_store.go

```diff
@@ -193,6 +193,10 @@ func (store *MongodbStore) ListDirectoryEntries(ctx context.Context, dirPath uti
 	optLimit := int64(limit)
 	opts := &options.FindOptions{Limit: &optLimit, Sort: bson.M{"name": 1}}
 	cur, err := store.connect.Database(store.database).Collection(store.collectionName).Find(ctx, where, opts)
+	if err != nil {
+		return lastFileName, fmt.Errorf("failed to list directory entries: find error: %w", err)
```

Comment:
Use %v ?

Reply:
Thank you for the attention to my pull request.
I guess it's better to use `%w` directive since go 1.13. Directive `%w` are similar to `%v`, but error returned with `%w` will have an `Unwrap` method. And also allows to use `errors.Is` and `errors.As` functions.
https://go.dev/blog/go1.13-errors

Reply:
Good to learn from this! Thanks!